test/suite/test_prepare_discover03.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_prepare_discover03.py
+#   Test that prepare discover cursor should return an error when closed with unclaimed prepared transactions
+
+import wiredtiger
+from suite_subprocess import suite_subprocess
+import wttest
+from wtscenario import make_scenarios
+
+class test_prepare_discover03(wttest.WiredTigerTestCase, suite_subprocess):
+    tablename = 'test_prepare_discover03'
+    uri = 'table:' + tablename
+    conn_config = 'precise_checkpoint=true,preserve_prepared=true'
+
+    types = [
```

Comment:
no need to use scenarios if we only have one?

Reply:
there is ticket to extend the testing. It is required to support column store in this project but I feel if we are able to support that, we can get better test coverage.